fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/util/InductionLoop.java

```diff
@@ -110,13 +110,5 @@ private double calculateTrafficFlow(long currentTime) {
         return duration > 0 ? total / (duration / (double) TIME.SECOND) * 3600d : 0;
```

Comment:
Not when we used this. Since a few versions of SUMO this [seems to be available](https://github.com/eclipse-sumo/sumo/issues/12030) via `getIntervalNumber`. But even then you need to know the size of the interval for normalization to `veh/h`. But even then, these intervals are fixed and are no floating windows like our implementation here (RingBuffer).

Reply:
Alright, I guess we'll leave it at that.